pyctbgui/utils/numpyWriter/npy_writer.py

```diff
@@ -0,0 +1,183 @@
+"""
+Wrapper to be able to append frames to a numpy file
+
+numpy header v1
+
+- 6bytes                         \x93NUMPY
+- 1 byte major version number    \x01
+- 1 byte minor version number    \x00
+- 2 bytes (unsigned short) HEADER_LEN length of header to follow
+- Header as an ASCII dict terminated by \n padded with space \x20 to make sure
+we get len(magic string) + 2 + len(length) + HEADER_LEN divisible with 64
+Allocate enough space to allow for the data to grow
+"""
+
+import ast
+import logging
+import os
+import zipfile
```

Comment:
How is this used? Only valid for npz files?

Reply:
i have made a small usage.md file [link](https://github.com/slsdetectorgroup/pyctbgui/blob/feature/numpyWriter-Module/pyctbgui/utils/numpyWriter/usage.md) it explains what can this library do